stenotype/Makefile

```diff
@@ -78,15 +78,15 @@ clean:
 
 # Generate g++ object files.
 %_opt.o: %.cc $(DEPS)
-	$(CXX) $(CFLAGS) -c -o $@ $<
+	$(CXX) $(CFLAGS) $(CPPFLAGS) $(SHARED_CFLAGS) -c -o $@ $<
```

Comment:
I'm not understanding this.  Up on line 59, we create CFLAGS from SHARED_CFLAGS.  Why do we re-add it here?

Reply:
True, this is indeed redundant. Removing it here.